compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -81,7 +82,7 @@ public fun <T : Component> SwingPanel(
 
     Box(
         modifier = modifier.onGloballyPositioned { coordinates ->
-            val location = coordinates.localToWindow(Offset.Zero).round()
```

Comment:
It's still not fully correct - it doesn't properly calculate the size. `boundsInWindow` should be used.
I already fixed in some time ago in #915 (and resolved conflicts after this PR)